apps/compounds_tester/assets/tailwind.config.js

```diff
@@ -13,7 +13,44 @@ module.exports = {
   ],
   theme: {
     extend: {
+      spacing: {
+        gap: "16pt",
```

Comment:
What's the advantages of using pt over rem or px?

Reply:
I asked myself the same thing. I was just going by what Geist had in this case